db/schema.rb

```diff
@@ -13,6 +13,9 @@
 
 ActiveRecord::Schema.define(version: 20141202222312) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
I think this change got added by PostgreSQL when I created the database on my computer.